pytests/helpers/session.py

```diff
@@ -42,6 +42,9 @@ def __init__(self, method='launch', ptvsd_port=None, pid=None):
         self.debug_options = ['RedirectOutput']
         self.env = os.environ.copy()
         self.env['PYTHONPATH'] = PTVSD_SYS_PATH
+        self.cwd = None
+        self.expected_returncode = 0
+        self.cli_args = ()
```

Comment:
Let's call this `program_args`, so that it's clear that these are what gets passed to the script/module.

Reply:
And also make it an empty list, so that the tests can `+=` things to it.